doc/nrf/releases/release-notes-1.1.0.rst

```diff
@@ -337,7 +337,7 @@ nRF BLE Controller
 
 * Improved the default static memory pool allocation.
   The controller now determines its static memory pool size based on the maximum Link Layer packet length.
-  This is determined by the Kconfig macro :option:`CONFIG_BT_CTLR_DATA_LENGTH_MAX` (if defined), or else the minimum packet length (which is 27 B).
+  This is determined by the Kconfig macro :kconfig:`CONFIG_BT_CTLR_DATA_LENGTH_MAX` (if defined), or else the minimum packet length (which is 27 B).
```

Comment:
I believe we should not change the old release notes. These are valid only in the context of older releases.

Reply:
It is sometimes necessary if, e.g. a Kconfig option is removed. In this case, it is necessary to make this Kconfig reference work.

Reply:
Sure but that will work only from NCS1.7 context and not for the release notes prepared for earlier versions.
I am wondering why we modify this files in the first place. This should not be part of processing for current release.

In the latest release notes things should obviously change. I am talking only about the old files.

@greg-fer can you tell if this allowed and if so why it is done?

Reply:
I had only very quick look on the release notes changes. There are changes related to wording but there are also renames of config options and changes of documentation reference. It's simply because old release notes are re-rendered using new versions of files. And these may again show things that are imprecise or even wrong from the older context.

We could fix it as you say by adding some context. For changes in options this may work but it won't work for links. We would need to make sure re-rendered release notes still point to the right places.

E.g.
Original NCS1.4.0 release notes (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.4.0/nrf/releases/release-notes-1.4.0.html) in section `BSD library` point to 
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.4.0/nrfxlib/bsdlib/doc/CHANGELOG.html#bsdlib-changelog
while the new version points to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_modem/doc/CHANGELOG.html#nrf-modem-changelog
In this case it's only adding more information but it might as well change information inside like in case of NCS1.3.0 release notes that depending how you look at them point to either https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_bootloader_external_flash.html#ug-bootloader-external-flash
or https://developer.nordicsemi.com/nRF_Connect_SDK/doc/1.3.0/nrf/ug_bootloader.html#ug-bootloader-flash

Reply:
@pdunaj , you are right we should not update the old release notes with new content after they are published. We always avoided that. However, we did update the old release notes on latest with formatting changes if changes to the doc framework required it. If there are changes that affect option names, these should be documented in the release notes for the release where the change happened.

Reply:
> we did update the old release notes on latest with formatting changes if changes to the doc framework required it

This is the source of the problem. The old release notes are re-rendered. This leads to need to follow new doc framework rules which is no problem in itself. The problem is that all links (option names, pages) are taken from the new repository as mentioned above.